Bio/SeqIO/AbiIO.py

```diff
@@ -354,7 +354,7 @@ def AbiIterator(handle, alphabet=None, trim=False):
         # handle empty file gracefully
         return
     if marker != b"ABIF":
-        raise IOError('File should start ABIF, not %r' % marker)
+        raise IOError('File should start with ABIF, not %r' % marker)
```

Comment:
I think the original is OK here, and the improved wording does not justify changing the exception and the small chance of breaking someone's script it is matched on the message.